speclang_modern.tex

```diff
@@ -2099,6 +2099,40 @@ \subsection{Non-terminating functions}
 to always terminate, which is the expected behavior of most
 functions.
 
+The \lstinline|terminates| clause can also be specified in a statement contract
+with the same meaning. When such a clause is not specified, it inherits the
```

Comment:
I'm not a fan of the expression *an ACSL clause is (or is not) specified* (it was never used in the document) since the purpose of the document is to specify the semantics of all ACSL clauses (by the way, all ACSL clauses are specified). 
I recommend this reformulation :
- First sentence -> *The termination of statements can also be specified in statement contracts by the same clause.*
- *When such a clause is not specified ...* -> *When such a clause is omitted ...*

Reply:
Done (not pushed)